### PR TITLE
feat: Button add button size and isBorder

### PR DESCRIPTION
**Pull request by jobsofferings**
Button add button size and isBorder

image

image

### CODE REVIEW

File: components/Button/src/index.svelte [hunk 1]
```diff
@@ -11,6 +11,8 @@
 	export let attrs = {}
 	export let type: IKunTypePro = 'primary'
 	export let disabled = false
+  export let size: 'normal' | 'small' | 'large' = 'normal'
```
Comment (baiwusanyu-c) [inline components/Button/src/index.svelte:14]: I don't really want to use normal small and large as the value of size, can you refer to this comment?
image

File: components/Button/__test__/button.spec.ts [hunk 2]
```diff
@@ -155,6 +155,31 @@ describe('Test: KButton', () => {
 		expect(host.innerHTML).matchSnapshot();
 	});
 
+	test('props: button size', async () => {
```
Comment (baiwusanyu-c) [inline components/Button/__test__/button.spec.ts:158]: Unit tests should be as comprehensive as possible.
Cover all three sizes.

**Review by baiwusanyu-c — CHANGES_REQUESTED** [review]

File: components/Button/__test__/button.spec.ts (hunk 2 above)
Comment (jobsofferings) [reply components/Button/__test__/button.spec.ts:158]: OK👌

File: components/Button/src/index.svelte (hunk 1 above)
Comment (baiwusanyu-c) [reply components/Button/src/index.svelte:14]: You only need to modify the name of the props and the name of the related class. Don’t refer to it for the value of the style, just design it reasonably.

**Comment by baiwusanyu-c** [comment]
Do you have any other changes to commit?

**Comment by jobsofferings** [comment]
Not yet?

**Pull request merged**